signup.css

```diff
@@ -108,8 +108,14 @@
     border-radius: 3px;
     text-align: center;
 }
+.login{
+    padding: 0rem;
+}
+.forgot-password{
+    padding: 0rem;
+}
 @media only screen and (max-width: 1024px){
-    .signup{
+    .authentication{
         grid-template-columns: 100%;
         grid-template-rows: auto;
     }
```

Comment:
is this a good practice to use a grid like this? plz look into it...  even I'm trying to find a better way

Reply:
I think it is a good practice.